src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -431,6 +432,13 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 *******************************************
 -->
 
+<PropertyGroup Condition="'$(AndroidBuildApplicationPackage)' == 'True'">
```

Comment:
That's a good catch. Adding the condition (in slightly different expression that the targets file uses everywhere else).